cross-media-measurement/src/main/kotlin/org/wfanet/measurement/kingdom/deploy/gcloud/spanner/readers/ScheduleReader.kt

```diff
@@ -58,7 +58,7 @@ class ScheduleReader : SpannerReader<ScheduleReader.Result>() {
       reportConfigId = struct.getLong("ReportConfigId"),
       scheduleId = struct.getLong("ScheduleId"),
       reportConfigDetails =
-        struct.getProtoMessage("ReportConfigDetails", ReportConfigDetails.parser())
+      struct.getProtoMessage("ReportConfigDetails", ReportConfigDetails.parser())
```

Comment:
This formatting looks bad even if it's technically right. How about adding a Struct.getReportConfigDetails private method?

```
private fun Struct.getReportConfigDetails(): ReportConfigDetails {
  return getProtoMessage("ReportConfigDetails", ReportConfigDetails.parser())
}
```